src/main/frontend/src/stores.js

```diff
@@ -0,0 +1,20 @@
+import { writable } from 'svelte/store';
+
+export function createLocalStore(key, initialValue) {
+	const localValue =
+		process.browser && !!localStorage.getItem(key) ? localStorage.getItem(key) : initialValue;
```

Comment:
I think this fixes a bug with theming in all Vlingo UI's, note that.

Reply:
Yeah... I tought it was svelte-materialify bug earlier 😁